src/common-utils/devcontainer-feature.json

```diff
@@ -54,5 +54,8 @@
             "default": false,
             "description": "Add packages from non-free Debian repository?"
         }
+    },
+    "containerEnv": {
+        "DEV_CONTAINERS_DIR": "/usr/local/etc/vscode-dev-containers"
```

Comment:
For future context, why do we need this?

Reply:
removed that.